env/include/defs.yml

```diff
@@ -33,24 +34,28 @@ ref_tk-flame_context_menu: [{name: ShotGrid Panel..., app_instance: tk-multi-sho
 
 ref_tk-flame-review:
   location:
-    version: v1.4.1
+    version: v1.4.2
     type: app_store
     name: tk-flame-review
   task_template: ''
+  # for higher quality, but wont work in firefox.
```

Comment:
This line was removed at one point but it is not clear why. Is that still relevant?